Docs/Help/Test-ADOPSConnection.md

```diff
@@ -0,0 +1,45 @@
+---
+external help file:
+Module Name:
+online version:
+schema: 2.0.0
+---
+
+# Test-ADOPSConnection
+
+## SYNOPSIS
+Test if an active connection is established to Azure DevOps.
+
+## SYNTAX
+
+```
+Test-ADOPSConnection [<CommonParameters>]
+```
+
+## DESCRIPTION
+Test if an active connection is established to Azure DevOps.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Test-ADOPSConnection
+```
+
+Test if an active connection is established to Azure DevOps.
```

Comment:
Please change the description to actually say what it returns. Something like :
```
Returns $true if connection....
Returns $false if...
```

Reply:
Added!